CHANGELOG.next.asciidoc

```diff
@@ -96,6 +96,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update haproxy.* fields to map to ECS. {pull}10558[10558] {pull}10568[10568]
 - Collect all EC2 meta data from all instances in all states. {pull}10628[10628]
 - Migrate docker module to ECS. {pull}10927[10927]
+- Fix for not reusable http client leading to connection leaks in Jolokia module
```

Comment:
```suggestion
- Fix for not reusable http client leading to connection leaks in Jolokia module {pull}11014[11014]
```

Reply:
Oh, and you have added it to the Breaking changes section, please add it to the bugfixes section.